src/pages/guides/theming/index.md

```diff
@@ -0,0 +1,397 @@
+
+```js
+// tailwind.config.js
+
+theme: {
+  extend: {
+    textColor: {
+      colorDefault: '#BB0000'
+    }
+  }
+}
+```
+
+If the text color in your Venia-themed app turns red, your configuration is working correctly. To test your own theme, refer to [Customizing your theme](https://tailwindcss.com/docs/adding-custom-styles#customizing-your-theme) from the Tailwind documentation.
+
+## Step 4: Customize the base styles
+
+You have two options for customizing the Tailwind and Venia base styles:
+
+1. Update the `tailwind.config.js` file directly.
+1. Import your own theme file as a `preset` (recommended).
+
+### Option 1: Update tailwind.config.js theme directly
+
+If you followed Step 3 above, you have already done this as a test. Another example: change the default font family by adding new styles to the `tailwind.config.js` `theme` property:
+
+```js
+// tailwind.config.js
+
+theme: {
+  extend: {
+    fontFamily: {
+      sans: ['"Open Sans"', 'sans-serif']
+    }
+  }
+}
+```
+
+You should avoid this option for all but the smallest changes. For example, if you plan to use the Venia base theme with only a few custom colors and fonts, this option may be fine for your needs. However, if you are going to make several style changes or plan to rotate through different themes (seasonal or otherwise), use option 2.
+
+### Option 2: Create your own theme presets
+
+We recommended this option for all but the smallest changes. The steps for creating your own [theme presets](https://tailwindcss.com/docs/presets) and adding them to the `tailwind.config.js` file to customize the existing base themes are as follows:
+
+1. Create a new `my-theme-preset.js` file anywhere in your project and name it whatever you want.
+1. Export a new `theme` object with your Tailwind and/or Venia base theme extensions and overrides:
+
+  ```js
+  // my-theme-preset.js
+
+  module.exports = {
+    theme: {
+      extend: {
+        fontFamily: {
+          sans: ['Courier', 'sans-serif']
+        }
+      }
+    }
+  }
+  ```
+
+1. Import your theme preset to the `tailwind.config.js` and add it to the `presets` array:
+
+  ```js
+  // tailwind.config.js
+
+  const venia = require('@magento/pwa-theme-venia');
+  const myThemePreset = require('./my-theme-preset');
+
+  module.exports = {
+    presets: [venia, myThemePreset]
+  }
+  ```
+
+  This example adds the `myThemePreset` after the `venia` preset, meaning that myThemePreset will be merged and override or extend the base styles from Venia and Tailwind. For more information, see [how configurations are merged](https://tailwindcss.com/docs/presets#merging-logic-in-depth) from the Tailwind documentation.
+
+1. Rebuild your app.
+
+### Override or extend Tailwind base styles
+
+To override or extend the Tailwind base styles for version `2.2.19`, and any other version, use this site as a [reference to the Tailwind base styles](https://unpkg.com/browse/tailwindcss@2.2.19/stubs/defaultConfig.stub.js).
```

Comment:
Can you say something like "the base styles for any supported version of TW"? Again, trying to future-proof the docs.